Python/Python.sublime-syntax

```diff
@@ -55,11 +223,21 @@ contexts:
         - match: \)
           scope: punctuation.definition.group.end.python
           pop: true
+        - match: ','
+          scope: punctuation.separator.tuple.python
+        - include: inline_for
         - include: expressions
     - match: \)
       scope: invalid.illegal.stray.brace.round.python
+    - match: \]
+      scope: invalid.illegal.stray.brace.square.python
+    - match: \}
+      scope: invalid.illegal.stray.brace.curly.python
     - include: lists
-    - include: dictionaries
+    - include: dictionaries_and_sets
+    # For debugging
+    # - match: \S
+    #   scope: invalid.illegal.unmatched
```

Comment:
Yes, kind of. It's a good place to look for unmatched "anything" since in the current syntax almost everything if not really everything gets matched by some pattern.